src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -1418,64 +1418,6 @@ SPECTRE_ALWAYS_INLINE constexpr const auto create_from(
 /// \endcond
 /**@}*/
 
-namespace DataBox_detail {
```

Comment:
So my plan was to use these runtime retrievals to do observing. However, with all the observer changes we've discussed, I'm not sure how relevant this code will be. The idea was that you could walk through a DataBox's tensor tags and retrieve it by name to as a `std::vector<DataVector>`. There might be better ways to do this nowadays, though.

I'm not asking for a change or to keep this code, just to give context for it so more people are aware of what I have in mind :)